plugins/metrics/src/main/java/org/apache/cloudstack/response/HostMetricsResponse.java

```diff
@@ -43,6 +43,10 @@
     @Param(description = "the total cpu allocated in Ghz")
     private String cpuAllocated;
 
+    @SerializedName("loadAverage")
```

Comment:
is cpuloadaverage better ?

Reply:
you are right, will change it to that

Reply:
For easier usage I changed both the results in 
list hosts
list hostsmetrics
to cpuloadaverage.